fix_shadow_fork.py

```
@@ -0,0 +1,12 @@
+with open("tests/test_shadow_fork.py", "r") as f:
```

Ruff (0.15.10)

[warning] Unnecessary mode argument

Remove mode argument

(UP015)